kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -270,29 +270,15 @@ define([
             }));
 
             stagingAreaViewer.$elem.append($fileTable);
-            stagingAreaViewer.$elem.find('table').dataTable({
+            
+            const fullDataTable = stagingAreaViewer.$elem.find('table').dataTable({
                 language: {
                     emptyTable: emptyMsg
                 },
                 dom: '<"file-path pull-left">frtip',
                 autoWidth: false,
                 order: [[4, 'desc']],
-                headerCallback: function (thead) {     
-                    function selectAllOrNone (event) {
```

Comment:
function moved below, as we need to wait until dataTables has finished rendering